packages/terra-functional-testing/src/commands/expect/toBeAccessible.js

```diff
@@ -0,0 +1,19 @@
+const { runAxe } = require('../axe');
+
+/**
+ * Invokes axe analysis on the current document and asserts there are no accessibility violations.
+ * @param {*} _ - Unused param. The expect is always run on the current document.
```

Comment:
Can you elaborate on this param? How is it used or why is it needed?

Reply:
expect requires a `received` argument

```
// Expect the result to equal expected.
expect(result).toEqual(expected);

// In this case the unused _ param would be the number 1.
expect(1).toEqual(2);
```

https://jestjs.io/docs/en/expect.html#expectextendmatchers

We run our expect against the entire page, we don't accept a parameter for this value.

```
// The expect takes no arguments, it is always run against the page.
expect().toBeAccessible();
```

Reply:
I see now. Thanks for the clarification!